src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -169,5 +180,8 @@ private static bool TryLoadAssembly(string assemblyFileName, ref Assembly assemb
 
             return false;
         }
+
+        private static string NormalizeAssemblyName(string assemblyName) =>
```

Comment:
Should we make this public?  We would then be able to use this in our repl middleware.

Reply:
This should be fine since this is more like a temporary solution.

Can you add a comment why need to normalize assembly name (due to roslyn running inside repl, etc...)